shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1172,6 +1172,7 @@ void WebContents::WebContentsCreatedWithFullParams(
 }
 
 bool WebContents::IsWebContentsCreationOverridden(
+    content::RenderFrameHost* opener,
```

Comment:
Looks like this was needed because of https://chromium-review.googlesource.com/c/chromium/src/+/6513812.